packages/flame/lib/src/game/flame_game.dart

```diff
@@ -18,52 +19,79 @@ import 'package:meta/meta.dart';
 ///
 /// This is the recommended base class to use for most games made with Flame.
 /// It is based on the Flame Component System (also known as FCS).
-class FlameGame extends ComponentTreeRoot
+class FlameGame<T extends World> extends ComponentTreeRoot
```

Comment:
nit: Can we make this generic a bit more descriptive? It is fine for methods to have single letter generics but reading`T _world` just feels so sad

I would say rename it to `GameWorld` or something 🤷

Reply:
If I read `GameWorld` I'm expecting a class probably though? And when I'm reading T I'm expecting something from generics, but I do get what you mean, I wonder what could be a good name :thinking:

Reply:
just `W` sounds good to me

Reply:
will this be a breaking change to any existing games that do not specify the generic type?

Reply:
Nope, it defaults to World. Changing to `W`.